learning/tour-of-beam/frontend/lib/pages/tour/screen.dart

```diff
@@ -0,0 +1,358 @@
+  Widget build(BuildContext context) {
+    return Column(
+      children: ['ParDo one-to-one', 'ParDo one-to-many']
+          .map((e) => _Unit(title: e))
+          .toList(),
+    );
+  }
+}
+
+class _Unit extends StatelessWidget {
+  final String title;
+  const _Unit({required this.title});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(bottom: BeamSizes.size18),
+      child: Row(
+        children: [
+          _ProgressIndicator(
+            assetPath: Assets.svg.unitProgress100,
+          ),
+          Text(title),
+        ],
+      ),
+    );
+  }
+}
+
+class _GroupTitle extends StatelessWidget {
+  final String title;
+  const _GroupTitle({required this.title});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        _ProgressIndicator(
+          assetPath: Assets.svg.unitProgress100,
+        ),
+        Text(
+          title,
+          style: Theme.of(context).textTheme.headlineMedium,
+        ),
+      ],
+    );
+  }
+}
+
+class _ProgressIndicator extends StatelessWidget {
+  final String assetPath;
+  const _ProgressIndicator({required this.assetPath});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(
+        left: BeamSizes.size4,
+        right: BeamSizes.size8,
+      ),
+      child: SvgPicture.asset(assetPath),
+    );
+  }
+}
+
+class _Content extends StatelessWidget {
+  const _Content();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: MediaQuery.of(context).size.height -
+          BeamSizes.appBarHeight -
+          TobSizes.footerHeight,
+      decoration: BoxDecoration(
+        color: ThemeColors.of(context).background,
+        border: Border(
+          left: BorderSide(color: ThemeColors.of(context).divider),
+        ),
+      ),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Expanded(
+            child: SingleChildScrollView(
+              controller: ScrollController(),
+              child: const FillerText(width: 1000),
+            ),
+          ),
+          const _ContentFooter(),
+        ],
+      ),
+    );
+  }
+}
+
+class _ContentFooter extends StatelessWidget {
+  const _ContentFooter();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(
+          top: BorderSide(color: ThemeColors.of(context).divider),
+        ),
+        color: ThemeColors.of(context).secondaryBackground,
+      ),
+      width: double.infinity,
+      padding: const EdgeInsets.all(BeamSizes.size20),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          Flexible(
+            child: OutlinedButton(
+              style: OutlinedButton.styleFrom(
+                primary: ThemeColors.of(context).primary,
```

Comment:
The recent Flutter has deprecated `primary` here.

Reply:
Done.